recipes/abseil/all/conanfile.py

```diff
@@ -258,9 +258,13 @@ def __init__(self, conanfile, filepath):
 
     def replace_in_options_file(self, options_filepath):
         for name, value in self.abi.items():
-            replace_in_file(self.conanfile, options_filepath,
-                    "#define ABSL_OPTION_{} 2".format(name),
-                    "#define ABSL_OPTION_{} {}".format(name, value))
+            replace_in_file(
+                self.conanfile,
+                options_filepath,
+                "#define ABSL_OPTION_{} 2".format(name),
+                "#define ABSL_OPTION_{} {}".format(name, value),
+                strict=False,
```

Comment:
```suggestion
```
Please, keep strict=True. It should replace a patch, which expects a specific case.

Reply:
The reason for putting strict=False is that I need the recipe to also work for versions that do not include a feature and so do not have a corresponding line for it in the file. What do you suggest, given that?